dashboard/src/main/java/org/openedx/dashboard/presentation/DashboardListViewModel.kt

```diff
@@ -160,6 +163,7 @@ class DashboardListViewModel(
                 if (isIAPFlow) {
                     iapNotifier.send(CourseDataUpdated())
                 }
+                pushNotifier.send(PushEvent.RefreshPushEvent)
```

Comment:
This will affect the atomic-ness of the method, better to create a dedicated function for this all call it from the same place we are calling this `updateCourses` method.

Reply:
dedicated method created.